packages/nimble-components/src/table-column/icon/tests/table-column-icon-matrix.stories.ts

```diff
@@ -0,0 +1,62 @@
+import type { StoryFn, Meta } from '@storybook/html';
+import { html, ViewTemplate } from '@microsoft/fast-element';
+import { createMatrixThemeStory } from '../../../utilities/tests/storybook';
+import { sharedMatrixParameters } from '../../../utilities/tests/matrix';
+import { Table, tableTag } from '../../../table';
+import { tableColumnIconTag } from '..';
+import { iconCheckTag } from '../../../icons/check';
+import { mappingIconTag } from '../../../mapping/icon';
+import { mappingSpinnerTag } from '../../../mapping/spinner';
+
+const metadata: Meta = {
+    title: 'Tests/Table Column Types',
+    parameters: {
+        ...sharedMatrixParameters()
+    }
+};
+
+export default metadata;
+
+const data = [
+    {
+        id: '0',
+        code: 0
+    },
+    {
+        id: '1',
+        code: 1
+    },
+    {
+        id: '2',
+        code: 2
+    }
+] as const;
+
+// prettier-ignore
+const component = (): ViewTemplate => html`
+    <${tableTag} id-field-name="id" style="height: 250px">
+        <${tableColumnIconTag}
+            field-name="code"
+            key-type="number"
+            group-index="0"
+        >
+            Column 1
+            <${mappingIconTag} key="0" label="Zero" icon="${iconCheckTag}" severity="success"></${mappingIconTag}>
+            <${mappingSpinnerTag} key="1" label="One"></${mappingSpinnerTag}>
```

Comment:
I was going to take a wait and see approach, but I guess it's silly to think it will be any different than the spinner stories. Unfortunately, we may have a problem disabling it. Because the spinner elements are dynamically created, I would have to add the workaround styling to the cell view template--I can't do it in the story itself. I also can't just add it to the table element and let it be inherited by the decendent spinner, because the spinner element's template sets the `--ni-private-spinner-animation-play-state` to `running` (thus overriding whatever I set on the table). I guess I'll go ahead and put it in the templates for the cell view and group header view. I can't think of any good reason that would be an issue.

Reply:
A couple possibilities:
1. add a (private undocumented) API to the mapping spinner element which sets the animation play state on the spinners it creates
2. use the `play` function in this story to query for spinners and set `--ni-private-spinner-animation-play-state` on them. (not sure if they'll be instantiated yet)

I agree it's a bad idea to call `isChromatic` from production code. While we actually have our own implementation of it so the import won't fail, it still doesn't seem ideal to add test-specific conditions to our components.

Reply:
I went for option 1. The only drawback is that it is paused in the matrix test while viewing in storybook, too--not just when taking Chromatic snapshots. That seems okay.